packages/runtime/container-runtime/src/summaryGenerator.ts

```diff
@@ -322,6 +327,8 @@ export class SummaryGenerator {
                         summarizeTelemetryProps = {
                             ...summarizeTelemetryProps,
                             clientSequenceNumber: summaryData.clientSequenceNumber,
+                            opsSizesSinceLastSummary: summaryData.opsSizesSinceLastSummary,
```

Comment:
Can we include it into IGeneratedSummaryStats (which is automatically included above via ...summaryData.summaryStats)? This would move where data is reported 2 stages up, but I think it's correct change (as this data is available / collected in "generate" state).

Less special treatment - the better :)

Reply:
Good point